Exercise_1.java

```diff
@@ -6,28 +6,57 @@
     boolean isEmpty() 
     { 
         //Write your code here 
+        if(a.length<=0)  

```

Comment:
when the stack is empty, it returns false which is incorrect. Check again and update the code